### PR TITLE
Update german translation as of v4.2.0

### PR DESCRIPTION
**Pull request by AndoKPD**

### CODE REVIEW

File: de.js [hunk 1]
```diff
@@ -84,36 +84,36 @@ settings.none Keine Einstellungen gefunden
...
 settings.local.lang Sprache
 settings.network.header Netzwerk
 settings.network.sendRate Hohe Tickrate
 settings.network.netRerouting Auto Routing (Beeinflusst Ping)
 settings.network.lagComp Lag Minimierung
-settings.network.hitReg Hitreg Test
+settings.network.hitReg Hitreg Test
 settings.popup.header Popups
-settings.popup.freeKRPop Free KR Popup anzeigen
-settings.experimental.header Experimental
```
Comment (NandrewBCannon) [inline de.js:97]: settings.experimental.header Versuchsweise

File: de.js [hunk 2]
```diff
@@ -36,12 +36,12 @@ generic.victory SIEG
 generic.matchover SPIEL VORBEI
 generic.matchaband SPIEL ABGEBROCHEN
 generic.follow Folgen
-generic.unfollow Nicht mehr folgen
+generic.unfollow Entfolgen
 generic.followers Follower
```
Comment (NandrewBCannon) [inline de.js:40]: Folger instead of follower

File: de.js [hunk 3]
```diff
@@ -177,7 +177,7 @@ settings.interface.oldBrowser Alter Server Browser
 settings.interface.rarityanim Seltenheits-Animationen ausschalten
 settings.interface.showPfp Profilbilder anzeigen
 settings.interface.showTradeMsg Trade-Nachrichten anzeigen
-settings.interface.hideNonTrade Hide Non-Tradable Skins
+settings.interface.hideNonTrade Vertecke nicht handelbare Skins
```
Comment (bodeneinheit) [inline de.js:180]: line 180 should be:
settings.interface.hideNonTrade Verstecke nicht handelbare Skins

**Review by bodeneinheit — APPROVED** [review]
everything fine

**Review by mty22 — APPROVED** [review]

**Comment by mty22** [comment]
kr queued

**Pull request merged**